crates/engine/tree/src/tree/invalid_block_hook.rs

```diff
@@ -43,11 +11,12 @@ pub struct NoopInvalidBlockHook;
 impl InvalidBlockHook for NoopInvalidBlockHook {
     fn on_invalid_block(
         &self,
+        _parent_header: &SealedHeader,
         _block: &SealedBlockWithSenders,
-        _header: &SealedHeader,
         _output: &BlockExecutionOutput<Receipt>,
         _trie_updates: Option<(&TrieUpdates, B256)>,
-    ) {
+    ) -> eyre::Result<()> {
```

Comment:
hmm, not a fan of returning a result here because this is a bit meaningless and results in more code in engine which just logs the error, but is fine

Reply:
agreed, also by handling errors inside the hook, we will have better logging with the origin of error